spec/features/search_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+RSpec.describe 'Food Search' do
+    describe 'happy path' do
+        it 'allows a user to search for foods' do
+            visit '/'
+            
+            fill_in "q", with: "sweet potatoes"
+            click_button("Search")
+
+            expect(current_path).to eq("/foods")
+        end 
+
+        it 'shows the number of returned results' do
+            visit '/'
+            
+            fill_in "q", with: "sweet potatoes"
+            click_button("Search")
+
+            expect(current_path).to eq("/foods")
+
+            expect(page).to have_content("Number of results: 49676")
```

Comment:
Typically I would warn against making assertions on exact values given that this data could change and break your test (if you're not mocking or using VCR to mock). For an assessment this is fine, but for exact value assertions, webmock or stubbing what comes back from a service call is a safer option for more stable testing.